src/CommandLineUtils/CommandLineApplication.cs

```diff
@@ -572,6 +580,11 @@ public CommandOption Option(string template, string description, CommandOptionTy
         /// </summary>
         /// <param name="option"></param>
         public void AddOption(CommandOption option)
+        {
+            AddOption((IOption)option);
+        }
+
+        internal void AddOption(IOption option)
         {
             _options.Add(option);
         }
```

Comment:
```suggestion
        public void AddOption(IOption option)
        {
            _options.Add(option);
        }
```

The cast and extra method hop appears unnecessary to me.